crates/voicevox_core/src/publish.rs

```diff
@@ -607,6 +610,44 @@ pub const fn error_result_to_message(result_code: VoicevoxResultCode) -> &'stati
     }
 }
 
+#[cfg(windows)]
+fn list_windows_video_cards() {
+    use std::{ffi::OsString, os::windows::ffi::OsStringExt as _};
+
+    use humansize::BINARY;
+    use tracing::{error, info};
+    use windows::Win32::Graphics::Dxgi::{
+        CreateDXGIFactory, IDXGIFactory, DXGI_ADAPTER_DESC, DXGI_ERROR_NOT_FOUND,
+    };
+
+    info!("利用可能なGPU (DirectMLには1番目のGPUが使われます):");
```

Comment:
利用可能かどうかはわからない（壊れてるとかありそう）なので、"検出された"のが良いかも？
```suggestion
    info!("検出されたGPU (DirectMLには1番目のGPUが使われます):");
```
（別にそのままでもほとんど問題なさそう！）